src/itdelatrisu/opsu/ui/NotificationManager.java

```diff
@@ -334,25 +338,37 @@ public void update(int delta) {
 
 	/**
 	 * Submits a bubble notification for drawing.
-	 * @param s the notification string
+	 * This method uses the default {@link Color#white} when drawing the notification borders
+	 * and does not have any associated {@link NotificationListener}s associated with it.
+	 * @param msg the notification string
+	 * @param args The formatting for the notification string, if any
 	 */
-	public void sendNotification(String s) { sendNotification(s, Color.white); }
-
+	public void sendNotification(String msg, Object... args) {
```

Comment:
Note: might want to overload all of these methods without the varargs. (not sure if possible, can check later)